src/ModAttachr.ts

```diff
@@ -5,7 +5,8 @@ import {
     ICallbackRegister, IEventCallback, IEventsRegister, IMod,
     IModAttachr, IModAttachrSettings, IMods, ITransformModName
 } from "./IModAttachr";
-
+

```

Comment:
Not sure what's happening with the endlines here. I'll file a note to update the git config to ignore them..